City-of-Helsinki/drupal-hdbt
# UHF-5289: Changes news theming to match the design

# News node styling [UHF-5289](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5289)

The existing styling for the news node was outdated and it has to be updated to match the current design

## What was done
* Styled teasers, fixed tag rendering on news node page, added "updated over a year ago", fixed social media share accessibility issue, added new paragraphs to be used in the news node

## How to install
* Make sure your frontpage is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the HDBT theme
    * `composer require drupal/hdbt:dev-UHF-5289-news-node-theming`
* Run `make drush-cr`

## How to test
* Go to some news page, or create a new one and check that the layout now matches the design. (For now the sidebar in wrongly on the left side but will be fixed in the region pr)
* The news item should also now have new paragraphs that can be added 

## Other PRs
* https://github.com/City-of-Helsinki/drupal-helfi-etusivu/pull/32
